### DISCUSSION TITLE
How to install MikTeX without showing installing progress bar windows by current user

### DISCUSSION THREAD

**Discussion by vmouwei** [MiKTeX/miktex#1124] [Q&A]
Is there a way to install miktex without showing progress bar windows by current user?
my command argumentlist as below:
$installArgumentList = "--unattended --private --user-install=$softwareInstallFolder"
Start-Process -FilePath $softwareInstallExe -Wait -WindowStyle Hidden -ArgumentList $installArgumentList

but alwarys shows progress bar windows, so do I miss some arguments? I want to this window no display.

Thanks

**Comment by vmouwei** [comment]


**Reply by arrowd** [reply]
What do I have to do with this?
